### PR TITLE
OpInfo: resize_, resize_as_

### PR DESCRIPTION
**Pull request by kshitij12345**
Reference: https://github.com/pytorch/pytorch/issues/54261

### CODE REVIEW

File: torch/testing/_internal/common_methods_invocations.py [hunk 1]
```diff
@@ -3676,6 +3676,27 @@ def generator():
...
+             )
+
+    def generator():
+        for shape, args_or_shape in cases:
+            if op_info.name == 'resize_':
+                args = (args_or_shape, )
+            elif op_info.name == 'resize_as_':
+                args = (make_arg(shape, requires_grad=False), )  # type:ignore[assignment]
+            else:
+                raise ValueError("sample_inputs_resize_ops is being used with incorrect operator")
```
Comment (mruberry) [inline torch/testing/_internal/common_methods_invocations.py:3693]: Nice catch

File: torch/testing/_internal/common_methods_invocations.py [hunk 2]
```diff
@@ -3676,6 +3676,27 @@ def generator():
...
+def sample_inputs_resize_ops(op_info, device, dtype, requires_grad, **kwargs):
+    make_arg = partial(make_tensor, dtype=dtype, device=device)
+    cases = (((S, S, S), (S * S, S)),
+             ((), ()),
+             ((), (1, 1, 1)),
+             )
+
+    def generator():
+        for shape, args_or_shape in cases:
+            if op_info.name == 'resize_':
```
Comment (mruberry) [inline torch/testing/_internal/common_methods_invocations.py:3688]: Future style note: comment here would be nice

File: torch/testing/_internal/common_methods_invocations.py [hunk 3]
```diff
@@ -3676,6 +3676,27 @@ def generator():
     return list(generator())
 
 
+def sample_inputs_resize_ops(op_info, device, dtype, requires_grad, **kwargs):
+    make_arg = partial(make_tensor, dtype=dtype, device=device)
+    cases = (((S, S, S), (S * S, S)),
+             ((), ()),
+             ((), (1, 1, 1)),
+             )
```
Comment (mruberry) [inline torch/testing/_internal/common_methods_invocations.py:3684]: Style nit (do not fix this): this paren is mismatched

**Review by mruberry — APPROVED** [review]
Cool! Nice work, @kshitij12345. I made a few notes to think about for the future but let's merge this as-is.

**Comment by mruberry** [comment]
Oh wait, this must have hit a merge conflict with tensor_split? I guess it does need an update, after all

**Comment by kshitij12345** [comment]
> Cool! Nice work, @kshitij12345. I made a few notes to think about for the future but let's merge this as-is.

I see that there is a conflict already. So is it ok to go ahead and resolve it and the nits?

**Comment by mruberry** [comment]
> > Cool! Nice work, @kshitij12345. I made a few notes to think about for the future but let's merge this as-is.
> 
> I see that there is a conflict already. So is it ok to go ahead and resolve it and the nits?

Yep!

File: torch/testing/_internal/common_methods_invocations.py (hunk 3 above)
Comment (kshitij12345) [reply torch/testing/_internal/common_methods_invocations.py:3684]: With
```python
cases = (((S, S, S), (S * S, S)),
         ((), ()),
         ((), (1, 1, 1)),
        )
```
we get
```
flake8 torch/testing/_internal/common_methods_invocations.py 
torch/testing/_internal/common_methods_invocations.py:3684:13: E124 closing bracket does not match visual indentation
```

Comment (mruberry) [reply torch/testing/_internal/common_methods_invocations.py:3725]: Huh; weird. Oh well

**Pull request closed without merging**